index.html

```diff
@@ -1015,9 +1015,16 @@ <h2>Versioning and Containment</h2>
     <section id="resource-authorization">
       <h2>Resource Authorization</h2>
       <p>
-        To configure access control restrictions, implementations MUST follow the recommendations of Web Access
-        Control [[!SOLIDWEBAC]] with the following additional requirements:
+        Implementations MUST follow the recommendations of Web Access Control [[!SOLIDWEBAC]] in configuring access
+        controls.
       </p>
+
+      <blockquote class="informative">
+        The [[!SOLIDWEBAC]] specification provides refinement on the original
+        <a href='https://www.w3.org/wiki/WebAccessControl'>Web Access Control wiki</a>. This section both adds further
```

Comment:
If we want to reference the WAC wiki then it should become a non-normative reference.

But is it really worth referencing? Or just confusing? -- I'd be inclined to omit it

Reply:
Agreed. If we just mentioned we're regining SOLIDWEBAC, that'd be enough.